yarn.lock

```diff
@@ -2366,98 +2366,118 @@
   optionalDependencies:
     npmlog "2 || ^3.1.0 || ^4.0.0"
 
-"@react-native-community/cli-clean@12.0.0-alpha.17":
-  version "12.0.0-alpha.17"
-  resolved "https://registry.yarnpkg.com/@react-native-community/cli-clean/-/cli-clean-12.0.0-alpha.17.tgz#1b612ef1538ea5feca2a5ad3b4184fa1c54f7538"
-  integrity sha512-ZE5hB1c2wQo50Np4l+Vnmn4GZehhV6GFoK/35SQTbVOIRSSxXs2S/bOLzlS4MLOC7486EODodfQXpcDZqzkG1g==
+"@react-native-community/cli-clean@12.0.0":
+  version "12.0.0"
+  resolved "https://registry.yarnpkg.com/@react-native-community/cli-clean/-/cli-clean-12.0.0.tgz#33f83709d566d1a59b317377eeb2a908b17509ae"
+  integrity sha512-wpR3317b18vQNAlAl8xa/+DA+3tX7gJj04dw6MWun2c6vk7o/iRCpk/FVbLpGx20k97ASW5fQ9reB2KJ+Wv7zg==
   dependencies:
-    "@react-native-community/cli-tools" "12.0.0-alpha.17"
+    "@react-native-community/cli-tools" "12.0.0"
     chalk "^4.1.2"
     execa "^5.0.0"
-    prompts "^2.4.2"
 
-"@react-native-community/cli-config@12.0.0-alpha.17":
-  version "12.0.0-alpha.17"
-  resolved "https://registry.yarnpkg.com/@react-native-community/cli-config/-/cli-config-12.0.0-alpha.17.tgz#e2c1489d01e0e3ca5e68907d6ad5dee78b6f5ef0"
-  integrity sha512-BzL6ix++hlGH3RibZhsDIB7B5tF8GlTeDrspg4FYw9+7uS2YqRNCQsemiCGm56gnftwFLCMbCqbiZMjaAVHtvg==
+"@react-native-community/cli-config@12.0.0":
+  version "12.0.0"
+  resolved "https://registry.yarnpkg.com/@react-native-community/cli-config/-/cli-config-12.0.0.tgz#ac6ae3cc409be52f3a40971331a5960a54d5977d"
+  integrity sha512-xGkqD7VtcAiDhI6pLXigJqGrd9voGPl+eQAhOvWWr1eZN7FfHM+jLhDI+JLDa6b3SNbFJBCXgiBunB6v90giWw==
   dependencies:
-    "@react-native-community/cli-tools" "12.0.0-alpha.17"
+    "@react-native-community/cli-tools" "12.0.0"
     chalk "^4.1.2"
     cosmiconfig "^5.1.0"
     deepmerge "^4.3.0"
     glob "^7.1.3"
     joi "^17.2.1"
 
+"@react-native-community/cli-debugger-ui@12.0.0":
+  version "12.0.0"
+  resolved "https://registry.yarnpkg.com/@react-native-community/cli-debugger-ui/-/cli-debugger-ui-12.0.0.tgz#949e48f1770414d5d3c08d14ddd5b5847b6034a4"
+  integrity sha512-gOid9bGi9dfGm+Ro89SFY9gZfrEk29MFn8wETgEGZ3K+/lelGzysfZmXyV0qk/N5nNurL3jOyhHRvLqU+XGOdQ==
+  dependencies:
+    serve-static "^1.13.1"
+
 "@react-native-community/cli-debugger-ui@12.0.0-alpha.17":
```

Comment:
We should figure out why an alpha version is still here. Like wise for `@react-native-community/cli-server-api`.

Reply:
Because `community-cli-plugin` right now also depends on CLI packages, and I forgot to change it there 😅 Thanks for catching this!